src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -194,14 +194,29 @@ public async Task<Manifests> DeserializeExistingManifestsAndUpdate(List<string>
                 return null;
             }
 
-            if (!PackageParser.UpdateInstallerNodes(installerManifest, this.InstallerUrls, packageFiles, out bool installerMismatch, out WingetCreateCore.Models.Installer.Installer installersMissingMatch))
+            if (!PackageParser.UpdateInstallerNodesAsync(
+                installerManifest,
+                this.InstallerUrls,
+                packageFiles,
+                out bool installerMismatch,
+                out List<WingetCreateCore.Models.Installer.Installer> unmatchedInstallers,
+                out List<WingetCreateCore.Models.Installer.Installer> multipleMatchedInstallers))
             {
                 if (installerMismatch)
                 {
                     Logger.ErrorLocalized(nameof(Resources.MultipleInstallerUpdateDiscrepancy_Error));

```

Comment:
We shouldn't log this error here, as we've already verified that the installer URL count matches (my bad)

Reply:
Removed logging here